Sources/protoc-gen-grpc-swift/main.swift

```diff
@@ -103,15 +102,20 @@ func main() throws {
   // Build the SwiftProtobufPluginLibrary model of the plugin input
   let descriptorSet = DescriptorSet(protos: request.protoFile)
 
+  // We need to generate conformance to `GRPCPayload` for request/response types. Track which
+  // types we've seen to avoid generating the conformance multiple times.
+  var observedMessages = Set<String>()
+
   // process each .proto file separately
   for fileDescriptor in descriptorSet.files {
```

Comment:
Is this ordering stable? Otherwise the location of the declaration might shift around.

Reply:
Good question; I honestly have no idea. It's certainly stable when the input is stable so I assume it depends on the order you pass them to protoc. Do you think it matters?

I'm not super happy with this approach but I'm not sure of a better one. The alternative I considered was generating the extensions based on the message descriptions (instead of the input/output for each RPC), that would make the location stable but it means you can generate conformance for messages which aren't used for RPCs. We'd also need to provide conformance for the "built in" messages (e.g. Any, Timestamp, etc) within `GRPC` so we'd be forced to update if SwiftProtobuf adds new messages.

Reply:
The problem with an unstable ordering would be that every `protoc` run could create different files, resulting in a lot of churn. How about simply sorting the list ourselves in some stable fashion to be sure? Just an idea, though.

Reply:
Sorting by filename is straightforward to add, so might as well.